double-check-tmp.md

```diff
@@ -0,0 +1,28 @@
+- No copyright/license mentioned
+  + http://www.jjj.de/fxt/fxtbook.pdf
+  + http://igm.univ-mlv.fr/~mac/REC/text-algorithms.pdf
+  + http://www.ethoberon.ethz.ch/WirthPubl/CBEAll.pdf
+  + http://lampwww.epfl.ch/~schinz/thesis-final-A4.pdf
+  + http://www.stack.nl/~marcov/compiler.pdf
+  + http://arxiv.org/pdf/1206.1754v2.pdf
```

Comment:
arxiv.org is fully open access and a well known scholarship repository. This one's fine.